docker/build-tools/Dockerfile

```diff
@@ -311,6 +323,7 @@ ENV GO111MODULE=on
 ENV GOPROXY=https://proxy.golang.org
 ENV GOSUMDB=https://sum.golang.org
 ENV GOPATH=/go
+ENV GOROOT=/go
```

Comment:
@sdake Didn't we have a problem with GOROOT in here before?

Reply:
The k8s code generation tools wouldn't work without this.